modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -1,7 +1,15 @@
-import { Arr, Obj, Optional } from '@ephox/katamari';
+import { Arr, Obj, Optional, Type } from '@ephox/katamari';
 import { Attribute, Compare, Css, CursorPosition, Insert, Replication, SelectorFilter, SugarElement, SugarNode, Traverse } from '@ephox/sugar';
 import { CellSpan, Generators, SimpleGenerators } from './Generators';
 
+const transferableAttributes: Record<string, string[]> = {
+  scope: [
+    'row',
+    'column',
```

Comment:
Turns out I was wrong about this sorry @HAFRMO. According to the spec it's only `col` that's allowed, not `column`. Sorry about that!